lib/fake-xhr/index.js

```diff
@@ -342,12 +342,27 @@ FakeXMLHttpRequest.parseXML = function parseXML(text) {
         try {
             if (typeof DOMParser !== "undefined") {
                 var parser = new DOMParser();
-                var parsererrors = parser
-                    .parseFromString("INVALID", "text/xml")
-                    .getElementsByTagName("parsererror");
-                var parsererrorNS = parsererrors.length
-                    ? parsererrors[0].namespaceURI : "";
-                var result = parser.parseFromString(text, "text/xml");
+                var parsererrorNS = "";
+
+                try {
+                    var parsererrors = parser
+                        .parseFromString("INVALID", "text/xml")
+                        .getElementsByTagName("parsererror");
+                    if (parsererrors.length) {
```

Comment:
In theory, this could probably be left out now as trying to read the `namespaceURI` from an empty array would be caught, but I somehow felt it's cleaner to keep it.